src/panels/lovelace/editor/config-elements/hui-entities-card-editor.ts

```diff
@@ -24,11 +24,11 @@ import "../../../../components/ha-icon";
 
 const entitiesConfigStruct = struct.union([
```

Comment:
Do we need this union? I can imagine it will be a lot easier for us to just convert it to an object before validating it ?

Reply:
Don't know if that is easier, would rather validate before processing it. Otherwise, we will still have to manual check to see if entities is present and if it is an array etc...

If we can just add this to the struct, I think it is easier this way. But until the editor supports all the functions we have to create custom ones :-(